typescript/vrotsc/src/compiler/transformer/fileTransformers/workflow/decorators/workflowItemDecoratorStrategy.ts

```diff
@@ -0,0 +1,93 @@
+/*-
+ * #%L
+ * vrotsc
+ * %%
+ * Copyright (C) 2023 - 2024 VMware
+ * %%
+ * Build Tools for VMware Aria
+ * Copyright 2023 VMware, Inc.
+ *
+ * This product is licensed to you under the BSD-2 license (the "License"). You may not use this product except in compliance with the BSD-2 License.
+ *
+ * This product may include a number of subcomponents with separate copyright notices and license terms. Your use of these subcomponents is subject to the terms and conditions of the subcomponent's license, as noted in the LICENSE file.
+ * #L%
+ */
+import { Decorator, MethodDeclaration, SourceFile } from "typescript";
+import { WorkflowItemDescriptor, WorkflowItemType } from "../../../../decorators";
+import { getDecoratorProps } from "../../../helpers/node";
+import CanvasItemDecoratorStrategy from "./canvasItemDecoratorStrategy";
+import { InputOutputBindings, buildItemParameterBindings } from "./helpers/presentation";
+import { findTargetItem } from "../helpers/findTargetItem";
+import { StringBuilderClass } from "../../../../../utilities/stringBuilder";
+
+export default class WorkflowItemDecoratorStrategy implements CanvasItemDecoratorStrategy {
+	getCanvasType(): string {
+		return "link";
+	}
+
+	getDecoratorType(): WorkflowItemType {
+		return WorkflowItemType.Workflow;
+	}
+
+	registerItemArguments(itemInfo: WorkflowItemDescriptor, decoratorNode: Decorator): void {
+		getDecoratorProps(decoratorNode).forEach((propTuple) => {
+			const [propName, propValue] = propTuple;
+			switch (propName) {
+				case "target":
```

Comment:
It is a good practice to wrap the cases with braces `{}`, i.e. 
```ts
case "target": {
    itemInfo.target = propValue;
    break;
}
```

Reply:
Really? Because I removed it because I thought it wasn't a good practice... I think It was also part of our convention that we don't do this

Reply:
I am not aware of such convention, however I think using braces would improve readability of the code.